src/lib/flow/Kconfig

```diff
@@ -4,8 +4,60 @@ config FLOW
 
 config RESOLVER_CONFFILE
 	bool "Resolver conffile"
+	depends on FLOW
 	default y
 
-config FLOW_RESOLVER
-	bool "Flow resolver"
+config NODE_DESCRIPTION
+	bool "Node description support"
+	depends on RESOLVER_CONFFILE
 	default y
+	help
+            Add node description support to enable runtime introspection.
```

Comment:
Be more descriptive here, Kbuild makes long descriptions nice to read. I'd say:

Keep meta information such as author, version, ports and options names in the binaries. This allows runtime to resolve a port index from its name or to parse options from text as it will know about the options, their names, types and requirements. These are essential for tool such as sol-fbp-runner or sol-fbp-generator.

The meta information will slightly increase the binary size, thus may be disabled for systems with constrained memory requirements. In this case only the sol-flow.h (low level C api) can be used as sol-flow-builder.h and FBP parsers are all dependent on this option.